recipes/gw_eccentricity/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "gw_eccentricity" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gw_eccentricity-{{ version }}.tar.gz
+  sha256: 90c5c9d4b8beaff26fd0e34414c1a41d6d9f68b8d1a79e07659d69765627cee8
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
```

Comment:
```suggestion
    - python >=3.6
    - pip
  run:
    - python >=3.6
```

Reply:
For missing packages it is fine to keep it noarch. That way the package is built only once and, if the missing dependency is ever built on Windows, this one would be ready for Windows already.

Reply:
I see. So it is okay if the checks do not pass for Windows for this PR? Also, in our package, the tests are only run for Linux and MacOS and therefore it's not really tested to work for Windows. Do you still think it should use `noarch: python`?

Reply:
> I see. So it is okay if the checks do not pass for Windows for this PR?

Yep.

> Also, in our package, the tests are only run for Linux and MacOS and therefore it's not really tested to work for Windows. Do you still think it should use noarch: python?

Unless you do know of anything that is platform specific it is OK. We save valuable CI resources with `noarch` packages.